recipes/libnetcdf/build.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+if [[ $(uname) == Darwin ]]; then
+  export LIBRARY_SEARCH_VAR=DYLD_FALLBACK_LIBRARY_PATH
+else
+  export LIBRARY_SEARCH_VAR=LD_LIBRARY_PATH
+fi
+
+export LDFLAGS="-L$PREFIX/lib $LDFLAGS"
+export CFLAGS="-I$PREFIX/include $CFLAGS"
+export CPPFLAGS="-I$PREFIX/include $CPPFLAGS"
+
+./configure --prefix=$PREFIX \
+            --enable-shared \
+            --enable-static \
+            --enable-extra-example-tests \
+            --disable-doxygen \
+            --enable-netcdf-4 \
+            --enable-ncgen4 \
+            --enable-dap \
+            --enable-hdf4
+
+make
+# https://www.unidata.ucar.edu/support/help/MailArchives/netcdf/msg11423.html
+eval ${LIBRARY_SEARCH_VAR}=$PREFIX/lib make check
```

Comment:
Could we please add `|| cat ncdump/test-suite.log` at the end of the `make check` line? This will let us see what is going on with the test failures.